ai-ml/gke-ray/rayserve/llm/tpu/ray-cluster.tpu-v5e-singlehost.yaml

```diff
@@ -0,0 +1,148 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_ai_ml_gke_ray_rayserve_raycluster_tpu_v5e_singlehost]
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  name: vllm-tpu
+spec:
+  headGroupSpec:
+    rayStartParams: {}
+    template:
+      metadata:
+        annotations:
+          gke-gcsfuse/volumes: "true"
+          gke-gcsfuse/cpu-limit: "0"
+          gke-gcsfuse/memory-limit: "0"
+          gke-gcsfuse/ephemeral-storage-limit: "0"
+      spec:
+        serviceAccountName: $KSA_NAME
+        containers:
+          - name: ray-head
+            image: $VLLM_IMAGE
+            imagePullPolicy: IfNotPresent
+            resources:
+              limits:
+                cpu: "8"
+                ephemeral-storage: 5Gi
+                memory: 40G
+              requests:
+                cpu: "8"
+                ephemeral-storage: 5Gi
+                memory: 40G
```

Comment:
**Question:** Is `memory: 40G` and `cpu: 8` necessary? Are we able to use less resources (to make this tutorial more approachable and less expensive)?

Reply:
We're not installing any additional dependencies not included in the image, so I think these can probably be lowered. I'll try it with `cpu: "2"` and `memory: "8G"` to validate this works first.

Reply:
Sorry for the delay @NimJay, I was finally able to scale up some slices and test with the lower resource requests. Everything works well from my manual testing so this PR should be good to merge whenever is convenient.